.travis.yml

```diff
@@ -1,3 +1,14 @@
+sudo: false
 language: node_js
-node_js:
-  - '6'
+cache:
+  directories:
+    - ~/.npm
```

Comment:
since we are doing `npm install`, it seems that it worth cache `node_modules` as well.

Reply:
Then it won't pick up the latest versions automatically (for range versioned deps). I don't think that's a good idea.